numpyro/examples/covtype.py

```diff
@@ -68,17 +68,20 @@ def benchmark_hmc(args, features, labels):
     t0 = time.time()
     # TODO: Use init_params from `initialize_model` instead of fixed params.
     hmc_state, _, _ = init_kernel(init_params, num_warmup_steps=0, step_size=step_size,
-                                  trajectory_length=trajectory_length, run_warmup=False,
-                                  adapt_step_size=False)
+                                  trajectory_length=trajectory_length,
+                                  heuristic_step_size=False, run_warmup=False)
+    sample_kernel(hmc_state.update(step_size=1.))
```

Comment:
Same comment here - I do not fully understand why this change is needed.

Reply:
I just want to compile `sample_kernel` separating from running `sampling`. (assume that it is compiled from warmup phase).

Reply:
Since `sample_kernel` isn't jitted here, I would be very surprised if this helps.

How about we keep the notebook for profiling purposes (where we can have hacks to ensure faster compilation, etc.), but have the examples scripts reflect how we expect users to use the HMC interface?